### ISSUE TITLE
Performance issue on client connection

### ISSUE THREAD

**Issue by neo-expert** [JPro-one/JPro-tickets#32]
When rendering in webrowser the ui freezes for a short time period if another client is connecting to the same application. You can reproduce it even with the official example:
https://demos.jpro.one/tilesfx.html
all the animations stuck for a short time, when someone else connects.

**Comment by FlorianKirmaier** [comment]
A very small freeze is expected, but it seems to be quite longer as it's supposed to be.

**Comment by FlorianKirmaier** [comment]
This seems to be a problem with the TilesFX sample.
The best way to test this is to open the Spinner form JFoenix.
When you open the ControlsFX sample, then you can't even see a short freeze.
That's how it should be ideally. 
The application shouldn't block the JavaFX-Thread, then there is no freeze.

**Comment by neo-expert** [comment]
Interesting thing is:  ControlsFX sample does stuck too, but only if someone connects to TilesFX. When the Application does a heavy background work, the UI freezes on all clients.

**Comment by FlorianKirmaier** [comment]
Correct,
We've found a performance-regression in JPro.
Your ticket was also useful for finding it.

We've already updated the website.
Now the freeze should be only a fourth compared to the old version.

The fix for the problem will probably be announced at the beginning of next week.

I'm closing the ticket now because it's not really a bug. These freezes can be avoided by not blocking the JavaFX-Thread during the start of an application.
